server/sonar-web/src/main/js/apps/issues/models/filter.js

```diff
@@ -21,7 +21,7 @@ import Backbone from 'backbone';
 
 export default Backbone.Model.extend({
   url () {
-    return '/api/issue_filters/show/' + this.id;
+    return window.baseUrl + '/api/issue_filters/show/' + this.id;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)